content/chainguard/libraries/cve-remediation.md

```diff
@@ -14,32 +14,64 @@ weight: 005
+protection against high and critical CVEs. Applications often rely on older
+versions of libraries, but upstream maintainers may not apply and release
+patches for those versions. CVE remediation addresses this gap by applying
+vulnerability fixes from upstream to older releases, particularly in cases where
+maintainers are no longer able to support and provide fixes.
 
-CVE Remediation helps reduce risk for organizations that cannot always upgrade quickly, without forcing disruptive dependency changes. This feature is included with [Chainguard Libraries for Python](/chainguard/libraries/python/overview). CVE Remediation is available for a subset of Chainguard Libraries for Python. If you want to request CVE Remediation for additional libraries, reach out to your account team.
+CVE remediation helps reduce risk for organizations that cannot always upgrade
+quickly since a larger upgrade to newer versions forces often disruptive
+changes. CVE remediation makes multiple incremental patch versions of affected
+older versions available and therefore allows a very minor upgrade that only
+addresses the CVE, but does not bring on other changes.
+
+CVE remediation is available for a subset of [Chainguard Libraries for
+Python](/chainguard/libraries/python/overview). If you want to request CVE
+remediation for additional libraries, reach out to your account team.
 
 ## About CVE Remediation
-CVE Remediation focuses on high and critical vulnerabilities across several Python libraries. Chainguard backports fixes that are already available in the upstream project to older versions that may no longer receive updates.
+
+CVE remediation focuses on high and critical vulnerabilities. Chainguard
+backports fixes that are already available in the new versions of the upstream
+project to older versions that may no longer receive updates.
 
 Before publishing a remediated version, Chainguard validates that the remediated version does not introduce regressions. All upstream test suites are run before and after applying the fix to confirm functional consistency. Chainguard also develops additional regression tests to validate the effectiveness of the CVE fix.
 
-Remediated libraries are distributed through a dedicated repository, giving you the option to choose when and whether to consume remediated versions.
+Remediated libraries are distributed through a dedicated repository. This
+provides the option to make remediated versions available for your development
+or opt out of using these versions complete and continue to use upstream
+versions only.
 
-To provide transparency, advisories for each CVE addressed in our remediated libraries are published via a public VEX feed
-at <https://libraries.cgr.dev/openvex/v1/all.json>. Supported scanners use this feed to identify and recognize remediated versions.
+Advisories for each CVE addressed in our remediated libraries are published via
+a public VEX feed at <https://libraries.cgr.dev/openvex/v1/all.json>. Supported
```

Comment:
Can we just make this a hyperlinked URL?

Reply:
Done